recipes/libdb/all/conanfile.py

```diff
@@ -0,0 +1,220 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, MSBuild, tools
+import os
+import shutil
+
+
+class LibdbConan(ConanFile):
+    name = "libdb"
+    description = "Berkeley DB is a family of embedded key-value database libraries providing scalable high-performance data management services to applications"
+    topics = ("conan", "gdbm", "dbm", "hash", "database")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.oracle.com/database/berkeley-db"
+    license = ("BSD", "LGPLv2", "Sleepycat")
```

Comment:
```suggestion
    license = ("BSD-3-Clause")
```

The license file only has a description which is BSD-3

Reply:
Thanks for noticing!